Patch to take CFLAGS, LDFLAGS from the users (Cons) environment.
===================================================================
--- gtk/makefile	(revision 34521)
+++ gtk/makefile	(working copy)
@@ -27,11 +27,11 @@
 THREADFLAGS=
 endif
 
 ifdef DEBUG
-CXXFLAGS=-DDEBUG -g $(CXXBASEFLAGS) $(THREADFLAGS)
+CXXFLAGS += -DDEBUG -g $(CXXBASEFLAGS) $(THREADFLAGS)
 else
-CXXFLAGS=-DNDEBUG -Os $(CXXBASEFLAGS) $(THREADFLAGS)
+CXXFLAGS += -DNDEBUG -Os $(CXXBASEFLAGS) $(THREADFLAGS)
 endif
 
 CONFIGFLAGS:=$(shell pkg-config --cflags gtk+-2.0)
 MARSHALLER=scintilla-marshal.o
